### DISCUSSION TITLE
Open XML create image from excel range

### DISCUSSION THREAD

**Discussion by ppiredda** [OfficeDev/Open-XML-SDK#1653] [General]
Hi all, sorry but i have this problem: I create an Excel file and after that I need to collect row and columns (range) and create an image in order to send it to the clients. I found SpreadSheetGear and it works but it is very expensive. Is it possibile to do that directly with open xml? Thanks a lot for your collaboration. Help mi please.

**Comment by AlfredHellstern** [comment]
Please ask this question on Stackoverflow. This is out of scope for the SDK.
